src/System.Management.Automation/engine/hostifaces/PSCommand.cs

```diff
@@ -63,14 +63,14 @@ internal PSCommand(Command command)
         #region Command / Parameter Construction
 
         /// <summary>
-        /// Add a command to construct a command pipeline.
+        /// Add a cmdlet to construct a command pipeline.
```

Comment:
Actually more than strict cmdlets can be used here. The path to a script will work as well, for example

Reply:
An overload for this method refers to `cmdlet` in its XML comments summary. Its parameter is also named `cmdlet` but as you mentioned elsewhere, this would be a breaking change.

Do you suggest to:

- Keep this the same and change the overload method's XML comments to also use `command` to connote that more than strict cmdlets can be used, or
- Allow the apparent inconsistency and make no changes?

Reply:
> Its parameter is also named cmdlet but as you mentioned elsewhere, this would be a breaking change.

Yes, it's an unfortunate inconsistency, but one that we must now live with. The parameter name is part of the API.

Feel free to add any extra documentation around the XML, but the parameter names themselves can't be changed.